.github/workflows/rust.yml

```diff
@@ -217,6 +217,10 @@ jobs:
           workspaces: |
             .
             examples/demo-prover
+      - name: Install cargo-risc0 # Risc0 v0.17 requires a cargo extension to build the guest code
+        run: cargo install cargo-risczero
```

Comment:
There's make command in Makefile, called `install-dev-tools`, could you please add it there, so we have single place to install all required dev tools?